### PR TITLE
Add JSqueezeFilter.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Filter/JSqueezeFilter.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ */
+class JSqueezeFilter extends AssetFilter
+{
+
+    /**
+     * Settings for JSqueeze minifier.
+     *
+     * @var array
+     */
+    protected $_settings = [
```
Comment (ADmad) [inline src/Filter/JSqueezeFilter.php:34]: No can do Mr. Stickler, the parent class has it so.

Comment (markstory) [inline src/Filter/JSqueezeFilter.php:34]: One day I'll fix that.

**Comment by markstory** [comment]
Can the `composer.json` suggest be updated?

**Comment by ADmad** [comment]
> Can the composer.json suggest be updated?

Done. I would also suggest removing jsmin from the suggests list and deprecating it's filter class. It's no longer maintained.

**Comment by markstory** [comment]
I can deprecate the jsmin filter.

**Comment by ADmad** [comment]
This is good to go then.

**Pull request merged**